.eslintrc

```diff
@@ -477,6 +477,7 @@
                     "workspace",
                     "writeable",
                     "xhr",
+                    "xlsx",
```

Comment:
I suppose it should be added with the usage of this word.

Reply:
I will include it next time, with all the xlsx code.